OpenDreamRuntime/Procs/Native/DreamProcNative.cs

```diff
@@ -9,6 +9,9 @@ static class DreamProcNative {
             DreamProcNativeRoot.MapManager = IoCManager.Resolve<IDreamMapManager>();
             DreamProcNativeRoot.ObjectTree = objectTree;
 
+            DreamProcNativeIcon.ObjectTree = objectTree;
+            DreamProcNativeMatrix.ObjectTree = objectTree;
```

Comment:
I think this is the right way to handle getting the ObjectTree into the native procs?

Reply:
Yep

Though in the future I want to avoid the static vars by storing them on the proc state (see #1065)